src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -236,11 +234,6 @@ export class VSCodeNotebookController implements Disposable {
         if (cells.length < 1) {
             return;
         }
-        if (this.isConnectionOutdated && !this.outdatedMessageDisplayed) {
-            void this.appShell.showErrorMessage(DataScience.jupyterRemoteConnectFailedModalMessage(), { modal: true });
```

Comment:
This is no longer required, we have better ways to handle this now.

Reply:
I.e. we let the connection crash and handle the errors in the usual central location `errorHandler.ts`